package.json

```diff
@@ -33,24 +33,28 @@
     "fastify": "~3.5.0",
     "fastify-autoload": "~3.3.0",
     "fastify-blipp": "~3.0.0",
+    "fastify-cors": "~4.1.0",
+    "fastify-helmet": "~5.0.1",
     "fastify-http-proxy": "~4.0.2",
+    "fastify-mongodb": "~2.0.1",
+    "fastify-no-icon": "~4.0.0",
     "fastify-plugin": "~2.3.3",
+    "graphiql": "~1.0.5",
+    "graphql-tools": "~6.2.0",
     "make-promises-safe": "~5.1.0",
+    "mercurius": "~6.3.0",
+    "mongoose": "~5.10.7",
```

Comment:
We are not going to use mongoose since it is really slow and it will become a major bottleneck soon. For the moment we can leave just the mongodb driver